
syntax = "proto3";

import "message.proto";

package bess.pb;

service FaaSControl {
	rpc UpdateFlow (FlowInfo) returns (FlowTableEntry) {}
}
